packages/mocks/src/console-handlers/providers.ts

```diff
@@ -0,0 +1,74 @@
+import { rest, RestHandler } from 'msw';
+
+import { createResource, ForkliftResourceKind } from '@kubev2v/legacy/client/helpers';
+import { IKubeList } from '@kubev2v/legacy/client/types';
+import { CLUSTER_API_VERSION } from '@kubev2v/legacy/common/constants';
+import { IProviderObject } from '@kubev2v/legacy/queries/types';
+
+import { MOCK_INVENTORY_PROVIDERS } from '../mocks/providers.mock';
+
+import { fancyCaptureUrl } from './util';
+
+const mockClusterProviders = (namespace?: string): IKubeList<IProviderObject> => {
```

Comment:
we have a PR to add mock data
a. the data will use auto generated types, that we know are up to date with latest API
b. the data should try to be static, e.g. one json file, so a user can "record" it from api calls using curl or just copy form the browser debug window

Reply:
ok, moved the the mock data PR